packages/codegen/src/generate.ts

```diff
@@ -79,6 +79,10 @@ export async function generateFiles(config: Config, dbMeta: DbMetadata): Promise
   const metadataFile: CodegenFile = {
     name: "./codegen/metadata.ts",
     contents: code`
+      ${setRuntimeConfig}({
+        temporal: ${config.temporal ? { timeZone: typeof config.temporal === "boolean" ? "UTC" : config.temporal.timeZone } : "false"},
+      });
```

Comment:
Seems likely we're going to want to reference `joist-config.json` data at runtime, so added this embedded-into-metadata.ts call to pass things along into a global `runtimeConfig` const.